core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -622,7 +622,7 @@ protected function row($strPalette=null)
 
 				$image = Image::getPath('placeholder.svg');
 
-				if ($blnCanResize)
+				if ($blnCanResize && $objFile->viewWidth && $objFile->viewHeight)
```

Comment:
Why did you add this check?

Reply:
Ok, then the following would be more clear IMHO:

```php
$blnCanResize = true;

if ($objFile->isSvgImage)
{
    // SVG images with undefined sizes cannot be resized
    if (!$objFile->viewWidth || !$objFile->viewHeight)
    {
        $blnCanResize= false;
    }
}
elseif (System::getContainer()->get('contao.image.imagine') instanceof Imagine)
{
    // Check the maximum width and height if the GDlib is used to resize images
    if ($objFile->height > Config::get('gdMaxImgHeight') || $objFile->width > Config::get('gdMaxImgWidth'))
    {
        $blnCanResize = false;
    }
}
```

WDYT?

Reply:
Yes, your version is way more clear! Can you commit directly to my PR?

Reply:
Done in ef19111f6ac915aff02ea640176679a94c042892.